dev_tools/packaging/set-dev-version.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+trap "{ echo -e '\033[31mFAILED\033[0m'; }" ERR
```

Comment:
These are causing surprising results when testing (my terminal closing unexpectedly).  These settings persist after running the script using `source`.